sqlflow/magic.py

```diff
@@ -45,11 +45,6 @@ def execute(self, line, cell):
         return self.client.execute('\n'.join([line, cell]))
 
 def load_ipython_extension(ipython):
-    out_handler = logging.StreamHandler(sys.stdout)
-    out_handler.setLevel(logging.INFO)
-    _LOGGER.addHandler(out_handler)
-    _LOGGER.setLevel(logging.INFO)
-
```

Comment:
@Yancey1989 Thanks for figuring out the cause for https://github.com/sql-machine-learning/sqlflow/issues/416!